### PR TITLE
added isPrime plugin to plugins.json

### PR DESCRIPTION
**Pull request by lvonkacsoh**
I hope everything's okay - first time creating a PR 😅

### CODE REVIEW

**Comment by jjw24** [comment]
hey thanks for the add, you forgot to put the repo as public.

**Comment by lvonkacsoh** [comment]
You are right! I fixed it

File: plugins.json [hunk 1]
```diff
@@ -201,6 +201,21 @@
...
+        "ID": "b9aaddfa36a948f384b31b4fb6186861",
+        "ActionKeyword": "isPrime",
+        "Name": "isPrime",
+        "Description": "Checks if the given arguments are prime numbers",
+        "Author": "lvonkacsoh",
+        "Version": "1.0.0",
+        "Language": "python",
+        "Website": "https://github.com/lvonkacsoh/Flow.Launcher.Plugin.IsPrime",
+        "IcoPath": "assets\\icon.png",
+        "ExecuteFileName": "main.py",
```
Comment (jjw24) [inline plugins.json:216]: could we please remove the attributes 'ActionKeyword' and 'ExecuteFileName', they are not needed.

File: plugins.json [hunk 2]
```diff
@@ -201,6 +201,21 @@
         "Website": "https://github.com/JohnTheGr8/Flow.Plugin.Github",
         "UrlDownload": "https://github.com/JohnTheGr8/Flow.Plugin.Github/releases/download/v1.2.2/Flow.Plugin.Github.zip",
         "UrlSourceCode": "https://github.com/JohnTheGr8/Flow.Plugin.Github/tree/main",
+        "E-tag": "W/\"6f4c4aa08e4286539f224e2af0cc37032c7a4c141e12ce7b65bd39639dab2695\""
```
Comment (jjw24) [inline plugins.json:204]: could you remove this E-tag and move this bottom e-tag: "E-tag": "W/\"b4aeb7a3c265e59c4a0736774bbaff98c0827b5758a4f9790d280d32d8b6932b\"" up here please.

E-Tag for the plugin be generated by the repo's CI automatically, so you dont need to add one for yours.

**Comment by jjw24** [comment]
I think it would be good if you upload the built package in Releases section of the repo, this will allow the manifest's CI to auto update its manifest with newer versions when you upload future releases.

image

**Comment by lvonkacsoh** [comment]
> I think it would be good if you upload the built package in Releases section of the repo, this will allow the manifest's CI to auto update its manifest with newer versions when you upload future releases.

I'm sorry, I am quiet inexperienced with GitHub so I didn't fully get what I should do. May I ask you to explain further how I can improve the plugin so your CI can update its manifest automatically?

**Comment by taooceros** [comment]
> I'm sorry, I am quiet inexperienced with GitHub so I didn't fully get what I should do. May I ask you to explain further how I can improve the plugin so your CI can update its manifest automatically?

Don't worry too much. You don't have to build the whole package since the plugin is written in python. All you need to do is to create a zip file that include everything of the plugin and attach that zip to the Release in github.

**Comment by jjw24** [comment]
Yeah no problem at all. The Releases page is one way of releasing your code for people to download, eg. if you look at Flow's release page, you can download the executable right, so for your plugin you can manually download the artifact from the url https://github.com/lvonkacsoh/Flow.Launcher.Plugin.IsPrime/archive/refs/tags/v1.0.zip and upload it to your initial release version. You can click on edit on the the release page to upload the artifact.

Like this:
image

We have a CI in this repo to automatically scan the plugin repos and then up the version and url so users will be able to download your newly published version and you don't have to make a PR here every time you release a new version. The code for the CI to update automatically is here:
https://github.com/Flow-Launcher/Flow.Launcher.PluginsManifest/blob/main/updateChecker.py

In the future if you are going to do more releases, you can consider doing a CI to have Github automatically package up and release your plugin.

**Comment by lvonkacsoh** [comment]
Thank you guys for the clarification and your patience! I added the artefact .zip file to the release and updated the manifest. Hopefully everything is good now :)

**Review by jjw24 — APPROVED** [review]

**Comment by jjw24** [comment]
All good to go, merged. Thanks for sharing

**Pull request merged**